### PR TITLE
Adkrawcz/log response dev

### PR DESCRIPTION
**Pull request by adamkrawczyk**
Added logging with rss logger

### CODE REVIEW

**Comment by adamkrawczyk** [comment]
Tests will be fixed soon

**Comment by lchojnack** [comment]
LGTM

File: ad_rss/src/core/RssCheck.cpp [hunk 1]
```diff
@@ -50,7 +53,7 @@ bool RssCheck::calculateProperResponse(world::WorldModel const &worldModel,
     if (!static_cast<bool>(mResponseResolving) || !static_cast<bool>(mSituationChecking)
         || !static_cast<bool>(mSituationExtraction))
     {
-      spdlog::critical("RssCheck::calculateProperResponse>> object not properly initialized");
+      mLogger->logCritical("RssCheck::calculateProperResponse>> object not properly initialized");
```
Comment (lchojnack) [inline ad_rss/src/core/RssCheck.cpp]: Capital letter: `object` -> `Object`

File: ad_rss/src/core/RssCheck.cpp [hunk 2]
```diff
@@ -65,17 +68,32 @@ bool RssCheck::calculateProperResponse(world::WorldModel const &worldModel,
...
       result = mResponseResolving->provideProperResponse(rssStateSnapshot, properResponse);
     }
+    // Send the message to the output
+    issueDescription = mLogger->getMessage();
   }
   // LCOV_EXCL_START: unreachable code, keep to be on the safe side
   catch (...)
   {
-    spdlog::critical("RssCheck::calculateProperResponse>> Exception caught");
+    mLogger->logCritical("RssCheck::calculateProperResponse>> exception caught");
```
Comment (lchojnack) [inline ad_rss/src/core/RssCheck.cpp]: Capital letter: `exception` -> `Exception`

File: ad_rss/src/core/RssResponseResolving.cpp [hunk 3]
```diff
@@ -203,10 +207,16 @@ void RssResponseResolving::combineState(state::UnstructuredSceneRssState const &
...
     responseHeadingRanges.clear();
     accelerationRange.maximum = std::min(accelerationRange.maximum, state.alphaLon.brakeMin);
+    mRssLogger_->appendMessage("RssResponseResolving::combineState>> Unstructured Scene, has to brake, max "
+                               "acceleration from accelerationRange: ",
+                               accelerationRange.maximum);
   }
   else
   {
     accelerationRange.maximum = std::min(accelerationRange.maximum, state.alphaLon.accelMax);
+    mRssLogger_->appendMessage("RssResponseResolving::combineState>> Unstructured Scene, don't has to brake, max "
```
Comment (lchojnack) [inline ad_rss/src/core/RssResponseResolving.cpp]: `don't have to brake`

File: ad_rss/src/core/RssSituationChecking.cpp [hunk 4]
```diff
@@ -100,7 +101,8 @@ bool RssSituationChecking::checkSituationInputRangeChecked(situation::Situation
     if ((!static_cast<bool>(mNonIntersectionChecker)) || (!static_cast<bool>(mIntersectionChecker))
         || (!static_cast<bool>(mUnstructuredSceneChecker)))
     {
-      spdlog::critical("RssSituationChecking::checkSituationInputRangeChecked>> object not properly initialized");
+      mRssLogger_->logCritical(
+        "RssSituationChecking::checkSituationInputRangeChecked>> object not properly initialized");
```
Comment (lchojnack) [inline ad_rss/src/core/RssSituationChecking.cpp]: Capital letter: `object` -> `Object`

**Review by lchojnack — COMMENTED** [review]
Minor suggestions. Otherwise LGTM.

**Comment by adamkrawczyk** [comment]
Added log response

**Pull request merged**